docs/make.jl

```diff
@@ -10,12 +10,16 @@ makedocs(;
         prettyurls = get(ENV, "CI", "false") == "true",
         canonical = "https://Pidoh.github.io/Pidoh.jl",
         assets = String[],
+        collapselevel = 1,
     ),
     pages = [
-        "Home" => "index.md",
-        "Problems" => "problems.md",
-        "Solution instances" => "instances.md",
-    ],
+        "Pidoh" => "index.md",
+        "Problems" => Any[
+            "bitstringproblems.md",
+            "problems.md",
+            ],
+        "Instances"=> "instances.md",
```

Comment:
You need to format the code using JuliaFormatter. you can do it either using JuliaFormatter [extension](https://github.com/domluna/JuliaFormatter.jl#editor-plugins) for your editor or by running `make format` in the root of the project.

Reply:
Yes! thanks. I didn't know about it. I've made a new commit.
Sorry for assigning before successful CI.